website/docs/d/iam_trusted_profiles_link.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profiles_link"
+description: |-
+  Get information about iam_trusted_profiles_link
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profiles_link
+
+Provides a read-only data source for iam_trusted_profiles_link. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
```

Comment:
1. Provides a read-only data source for iam_trusted_profiles_link.
as
Provides a read-only data source for IAM trusted profiles link.
2. reference link to refer to IAM trusted profiles link.